compiler/modules/front/parse.bal

```diff
@@ -2,9 +2,13 @@
 
 import wso2/nballerina.err;
 
-function parseModulePart(string str) returns ModulePart|err:Syntax {
```

Comment:
This changes is not an improvement IMHO

Reply:
I need a function that parses `ModulePart` given a `Tokenizer`
All the other `parse*` in this file accept a `Tokenizer tok`, So I though it would be more consistent if didn't create a different functions and change the signature of this to match others.